src/main.ts

```diff
@@ -161,6 +161,11 @@ export default class DataviewPlugin extends Plugin {
                 });
             })
         );
+
+        window.CodeMirror.defineMode("dataviewjs", config => window.CodeMirror.getMode(config, "javascript"));
```

Comment:
Might be worth noting that the codeblock-language should not be hardcoded, as the user can change it in the dataview settings (at least in theory, don't think there is a real reason to do so though)
![Pasted image 2024-05-16 at 16 16 52](https://github.com/blacksmithgu/obsidian-dataview/assets/73286100/5fe7f1d3-2990-4b3f-9cf2-cc113a1724a1)

Reply:
That's a very valid point. I'll make a fix for that